tox.ini

```diff
@@ -25,14 +25,6 @@ passenv =
         DOCKER_*
         DJANGO_TEST_PROCESSES
 
-commands_pre =
```

Comment:
Improve the commit message to clarify that the tox file was changed.

Reply:
Done.